index.js

```
@@ -1 +1,85 @@
 // implement your API here
+const express = require('express');
+const server = express();
+const port = 3001;
+const db = require('./data/db');
+server.use(express.json());
+
+server.get
+
+server.get('/', (req, res) => {
+    res.redirect('/api/users');
+});
+
+server.get('/api/users', (req, res) => {
+    db.find()
+    .then(users => {
+        res.status(200).json(users);
+    })
+    .catch(error => {
+        res.status(500).json(error)
```

Need to return the error message as written in the README. "The user information could not be retrieved".

---

Same with the ones below. Not a big deal since it's all functional, but it's good practice.